fairscale/experimental/nn/ssd_offload.py

```diff
@@ -90,22 +94,26 @@ class SsdTensorHandle(torch.Tensor):
     point in time the Tensor may be in memory or on disk.
 
     Args:
-        shape Tuple[int, ...]: Shape of the tensor that is represented by the handle.
+        shape torch.Size: Shape of the tensor that is represented by the handle.
         dtype: torch.dtype: Dtype of the tensor that is represented by the handle.
         requires_grad: bool: Property of the tensor that is represeneted by the handle.
 
     Returns:
         A SSDTensorHandle object representing a Tensor.
     """
 
+    override_directory_path: Optional[str] = None
```

Comment:
can you add a comment about why we need this class variable?

Reply:
Done in next version of PR.